crypto/bio/b_addr.c

```diff
@@ -565,7 +566,8 @@ static int addrinfo_wrap(int family, int socktype,
                          unsigned short port,
                          BIO_ADDRINFO **bai)
 {
-    OPENSSL_assert(bai != NULL);
+    if (bai == NULL)
```

Comment:
Side note: This actually gets interesting, 'cause while we're generally bashing assertions, not checking for NULL is the same as a hart assert of `ptr != NULL`...   food for thoughts...